### DISCUSSION TITLE
cx_freeze packaging tortoise-orm application

### DISCUSSION THREAD

**Discussion by HyperDev724** [marcelotduarte/cx_Freeze#2554] [Q&A]
Use cx_freeze to package the tortoise-orm application. After packaging is complete, running it will result in an error message stating that the tortoise-orm package cannot be found.
![image](https://github.com/user-attachments/assets/252928e0-6775-4b08-b8f8-71e121e308c0)

**Comment by marcelotduarte** [comment]
Please open an issue [bug report](https://github.com/marcelotduarte/cx_Freeze/issues/new?assignees=&labels=&projects=&template=bug-report.md&title=) with more details, that I'll create a hook.
